File: backend/booking-service/src/main/java/com/project/booking/dto/booking/BookingResponse.java
package com.project.booking.dto.booking;

import com.project.booking.model.enums.BookingStatus;
import java.math.BigDecimal;
import java.time.LocalDateTime;
import java.util.List;
import java.util.UUID;

public record BookingResponse(
        Long id,
        Long createdByUserId,
        UUID tourId,
        UUID tourPackageId,
        String bookingCode,
        Integer numberPassengers,
        String primaryContactName,
        String primaryContactPhone,
        BigDecimal totalAmount,
        BookingStatus status,
        List<BookingPassengerInfoResponse> bookingPassengers,
        LocalDateTime createdAt,
        LocalDateTime updatedAt) {
}

File: frontend-android/app/src/main/java/com/project/cruise/android/data/dto/booking/BookingResponse.kt
package com.project.cruise.android.data.dto.booking
import java.math.BigDecimal
data class BookingResponse(
    val id: Long,
    val createdByUserId: Long?,
    val primaryContactEmail: String?,
    val tourId: String?,
    val tourPackageId: String?,
    val bookingCode: String?,
    val numberPassengers: Int?,
    val primaryContactName: String?,
    val primaryContactPhone: String?,
    val totalAmount: BigDecimal?,
    val status: BookingStatus?,
    val bookingPassengers: List<BookingPassengerInfoResponse>?,
    val createdAt: String?,
    val updatedAt: String?
)